core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -182,6 +182,12 @@ package object config {
   private[spark] val SHUFFLE_SERVICE_ENABLED =
     ConfigBuilder("spark.shuffle.service.enabled").booleanConf.createWithDefault(false)
 
+  private[spark] val SHUFFLE_SERVICE_DB_ENABLED =
+    ConfigBuilder("spark.shuffle.service.db.enabled")
```

Comment:
what's the reason this is off by default?

Reply:
Hi，@rxin，There is no reason for that，I also  think it's better left on by default and I will fix it.